src/commonMain/kotlin/io/github/koalaplot/core/xygraph/XYGraph.kt

```diff
@@ -359,41 +385,64 @@ private data class ChartAreas(
      * Returns a copy of this ChartAreas after setting the [xAxisLabelAreaHeight],
      * based on the x-axis labels and their rotation angle.
      */
-    fun withComputedXAxisLabelAreas(xAxisLabels: List<Measurable>, rotation: Int): ChartAreas {
+    fun withComputedXAxisLabelAreas(
+        xAxisLabels: List<Measurable>,
+        rotation: Int
+    ): ChartAreas {
         val xAxisLabelHeights = xAxisLabels.calcXAxisLabelWidthConstraints(
             rotation,
             xTickSpacing(xAxisLabels.size),
             constraints.maxHeight - graphSize.height - xAxisHeight - xAxisTitleHeight + xAxisOffset,
             { meas, w -> meas.maxIntrinsicHeight(w) },
             { h -> h }
         )
-
         val labelAreas = xAxisLabels.mapIndexed { index, label ->
             RotatedComposableAreaDelegate(
                 IntSize(label.maxIntrinsicWidth(xAxisLabelHeights[index]), xAxisLabelHeights[index]),
                 rotation.toFloat()
             )
         }
-
         return copy(xAxisLabelAreaHeight = labelAreas.maxOfOrNull { it.height } ?: 0)
     }
 
     /**
      * Returns a copy of this ChartAreas after setting the [yAxisLabelAreaWidth]
      * based on the y-axis labels and their rotation angle.
      */
-    fun withComputedYAxisLabelAreas(yAxisLabels: List<Measurable>, rotation: Int): ChartAreas {
+    fun withComputedYAxisLabelAreas(
+        yAxisLabels: List<Measurable>,
+        rotation: Int
+    ): ChartAreas {
         val yAxisLabelAreas = yAxisLabels.map {
             val width = it.maxIntrinsicWidth(graphSize.height / yAxisLabels.size)
-
             RotatedComposableAreaDelegate(
                 IntSize(width, it.maxIntrinsicHeight(width)),
                 rotation.toFloat()
             )
         }
-
         return copy(yAxisLabelAreaWidth = yAxisLabelAreas.maxOfOrNull { it.width } ?: 0)
     }
+
```

Comment:
If you add the ```pointerInput``` modifier to the ```Box``` wrapping the ```chartScope.content()```, then I think none of this is needed.